schema.json

```diff
@@ -1 +1 @@
items":{"additionalProperties":true,"properties":{"keyword":{"additionalItems":false,"description":"List some keywords pertaining to the skill","items":{"additionalProperties":true,"properties":{"name":{"description":"e.g. HTML - [Add the skill name]","type":"string"},"score":{"description":"e.g. 20 - [Score for the skill name]","type":"number"}},"type":"object"},"type":"array"},"name":{"description":"e.g. Web Development","type":"string"}},"type":"object"},"type":"array"},"volunteer":{"additionalItems":false,"items":{"additionalProperties":true,"properties":{"endDate":{"description":"e.g. 2017-06-29 - [resume.json uses the ISO 8601 date standard] ","format":"date","type":"string"},"highlights":{"additionalItems":false,"description":"Specify accomplishments and achievements","items":{"description":"e.g Invited as a speaker in Xyzcon'17","type":"string"},"type":"array"},"location":{"additionalProperties":false,"description":"e.g. Germany - [Location for this activity]","format":"location","properties":{"lat":{"type":"number"},"long":{"type":"number"}},"type":"object"},"organization":{"description":"e.g. Xyz ","type":"string"},"position":{"description":"e.g. Open Source Contributor - [Contribution type]","type":"string"},"startDate":{"description":"e.g. 2014-06-29 - [resume.json uses the ISO 8601 date standard]","format":"date","type":"string"},"summary":{"description":"Give an overview of your responsibilities at the company","type":"string"},"url":{"description":"e.g. http://xyz.example.com - [Related link to support volunteer experience]","format":"uri","type":"string"}},"type":"object"},"type":"array"},"work":{"additionalItems":false,"items":{"additionalProperties":true,"properties":{"description":{"description":"e.g. A social media company - [Description of the companies primary activity]","type":"string"},"endDate":{"description":"e.g. 2018-12-29 - [resume.json uses the ISO 8601 date standard]","format":"date","type":"string"},"highlights":{"additionalItems":false,"description":"Specify multiple accomplishments","items":{"description":"e.g. Worked with mobile team at Twitter to develop remote debugging tools for mobile browsers","type":"string"},"type":"array"},"location":{"additionalProperties":false,"description":"e.g. Germany - [Location for this activity]","format":"location","properties":{"lat":{"type":"number"},"long":{"type":"number"}},"type":"object"},"name":{"description":"e.g. XYZ Inc. - [Company name]","type":"string"},"position":{"description":"e.g. Software Engineer - [Position at the company]","type":"string"},"startDate":{"description":"e.g. 2017-06-28 - [resume.json uses the ISO 8601 date standard]","format":"date","type":"string"},"summary":{"description":"Give an overview of your responsibilities at the company","type":"string"},"url":{"description":"e.g. http://xyz.example.com - [Related link to the company website]","format":"uri","type":"string"}},"type":"object"},"type":"array"}},"title":"Resumic Schema","type":"object"}
```

Comment:
In the long run we could think about solutions to make it more human readable as a json file. :smile:

Reply:
Awesome. I was too slow with the review :)